hpedockerplugin/hpe_storage_api.py

```diff
@@ -635,6 +638,17 @@ def volumedriver_create_snapshot(self, name, mount_conflict_delay,
                                                  has_schedule,
                                                  schedFrequency)
 
+    def generate_schedule_with_timestamp(self):
+        import datetime
```

Comment:
Please move import to the top of the module.

Reply:
Done